src/main/java/uk/gov/hmcts/reform/pip/data/management/utils/JsonExtractor.java

```diff
@@ -1,18 +1,23 @@
 package uk.gov.hmcts.reform.pip.data.management.utils;
 
+import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.jayway.jsonpath.Configuration;
 import com.jayway.jsonpath.DocumentContext;
 import com.jayway.jsonpath.JsonPath;
 import com.jayway.jsonpath.Option;
+import com.networknt.schema.JsonSchema;
+import com.networknt.schema.JsonSchemaFactory;
+import com.networknt.schema.SpecVersion;
+import com.networknt.schema.ValidationMessage;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.reform.pip.data.management.config.SearchConfiguration;
 
 import java.io.IOException;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
+import java.io.InputStream;
+import java.text.SimpleDateFormat;
+import java.util.*;
```

Comment:
I have a feeling checkstyles will complain about the '*' import here, so will need changing to be individual imports

Reply:
done